parmed/dlpoly/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
+Contains classes for parsing DLPOLY topology and parameter files
+"""
+import os as _os
+from parmed.utils import which as _which
+
+__all__ = ['DLPOLY_TOPDIR', 'DlpolyFieldFile', 'DlpolyConfigFile']
+
+DLPOLY_TOPDIR = None
+
+if _os.getenv('GMXDATA') is not None and _os.path.isdir(
+        _os.path.join(_os.getenv('GMXDATA'), 'top')):
+    DLPOLY_TOPDIR = _os.path.join(_os.getenv('GMXDATA'), 'top')
+elif _os.getenv('GMXBIN') is not None and _os.path.isdir(
+        _os.path.join(_os.getenv('GMXBIN'), '..', 'share', 'dlpoly', 'top')):
+    DLPOLY_TOPDIR = _os.path.join(_os.getenv('GMXBIN'), '..', 'share',
```

Comment:
Does this really use the GROMACS environment variables?

Reply:
Ops... I have removed these unnecessary calls. Good catch!